tests/output/ConsoleInteractionTest.py

```diff
@@ -189,6 +190,22 @@ def tearDown(self):
         OpenEditorAction.is_applicable = self.old_open_editor_applicable
         ApplyPatchAction.is_applicable = self.old_apply_patch_applicable
 
+    def test_color_letter(self):
+        line1 = '[  ] 1. (A)pply Patch'
+        with retrieve_stdout() as stdout:
+            color_letter(self.console_printer, line1)
+            self.assertEqual(line1 + '\n', stdout.getvalue())
+
+        line2 = '[  ] *0. Apply (P)atch'
+        with retrieve_stdout() as stdout:
+            color_letter(self.console_printer, line2)
+            self.assertEqual(line2 + '\n', stdout.getvalue())
+
+        line3 = '[  ] 3. Apply (P)atch [Note: This will do something]'
+        with retrieve_stdout() as stdout:
+            color_letter(self.console_printer, line3)
+            self.assertEqual(line3 + '\n', stdout.getvalue())
```

Comment:
I am not familiar with this part, but is it able to verify the color of output?

Reply:
No, it just checks the parsing of the line.

Reply:
I will look if there is a way to verify color as well

Reply:
I tried using `termcolor` for generating colored string but there is always a difference in some bits.